yo/app/scripts/services/DataSetService.js

```diff
@@ -1,3 +1,6 @@
+/*jshint undef: false, unused: false, indent: 2*/
```

Comment:
I added it only to the files I created/modified but probably it should be added everywhere.